### ISSUE TITLE
Repository name change

### ISSUE THREAD

**Issue by splorp** [UNNA/ikiwiki-newton_knowledge_base#2]
What do you think about changing the name of this repo, replacing the underscores with hyphens?

`ikiwiki-newton-knowledge-base`

Just for consistency with the other repos.

**Comment by morgant** [comment]
Yes, that sounds good. In fact, I think we should just rename it to the original `wikiwikinewt`, but I'd prefer to do renaming after I've found the ikiwiki config files amongst the backups.

**Comment by splorp** [comment]
That make complete sense.
